docs/windows.md

```diff
@@ -79,6 +79,10 @@ prefer running the Antrea Agent as a Windows service, or if you are using
 containerd 1.5, you can use the [Installation as a
 Service](#installation-as-a-service) method.
 
+Starting from v2.1, Antrea Windows image is built on Linux host with docker buildx
+and uses [hpc](https://github.com/microsoft/windows-host-process-containers-base-image)
+as the base image. This requires containerd 1.6.18 or higher.
```

Comment:
Is this requirement for _building_ the image or _running_ the image. If it is for building the image, it's not really relevant to users.

Reply:
This is required to import the image. According to CI test, after we build the image using buildx and save it locally,  jenkins job will scp the tar file to Windows Node. If the Containerd runtime on Windows Node is lower than 1.6.18, it may fail to import the image. So highlight this requirement here.

Reply:
That seems specific to our CI (or specific to building the image manually) and would just confuse users IMO.

Reply:
Got it, I will remove this requirement on containerd version.